tests/console/wicked_risky.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Test scenarios:
+# Test 1 : Create a gre interface from legacy ifcfg files
+# Summary: Collection of wicked tests which might be hard to implement in
+# openQA. Used as POC for wicked testing. Later will be reorganized in other
+# test suites.
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils qw(systemctl snapper_revert_system);
+
+
+sub run {
+    my ($self)   = @_;
+    my $gre_ipv4 = '';
+    my $gre_ipv6 = '';
+    type_string("#***Test 1: Create a gre interface from legacy ifcfg files***\n");
+    if (check_var('WICKED', 'RISKY_REF')) {
+        $gre_ipv4 = '10.0.2.10';
```

Comment:
you are repeating the ip addresses. Better define them in a perl package which you use in multiple test modules or define them in a base class.

Reply:
my concern here is  data/wicked/ifcfg-gre1_ref and data/wicked/ifcfg-gre1_sut. By moving this IP's  out from wicked_risky and wicked_before_test I will not solve the problem of "having same data duplicated in several places". In my opinion it not worth creation of another perl file which only purpose would be keep 2 IPs.

Reply:
Then grep it from these data files?